src/tree/CosmosDBAccountWizard/CosmosDBAccountCreateStep.ts

```diff
@@ -30,7 +30,7 @@ export class CosmosDBAccountCreateStep extends AzureWizardExecuteStep<ICosmosDBW
         const options: DatabaseAccountCreateUpdateParameters = {
             location: locationName,
             locations: [{ locationName: locationName }],
-            kind: defaultExperience.kind,
+            kind: defaultExperience.kind as DatabaseAccountKind,
```

Comment:
```suggestion
            kind: defaultExperience.kind,
```

I don't think this is needed either. Was typescript complaining about this?